projects/experimental/directives/button-group-card/button-group-card.style.less

```diff
@@ -0,0 +1,52 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiButtonGroupCard] {
+    display: flex;
+    min-width: 21.4375rem;
+    border-radius: 1.5rem;
+
+    &[data-surface='dark'] > [tuiButton][tuiAppearance] {
+        color: #fff;
+
+        &:hover {
+            background: rgba(256, 256, 256, 0.13);
+        }
+    }
+
+    > [tuiButton][tuiButtonVertical] {
+        flex-grow: 1;
+        flex-basis: 0;
+        border-radius: 0;
+        padding: 1.125rem 0.25rem;
+        min-width: 6.625rem;
+        gap: 0.5rem;
+    }
+
+    > [tuiButton][tuiButtonVertical]:only-child {
```

Comment:
I think it also has bigger font in the specs.

Reply:
what do you mean?

<img width="670" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/1bacca5d-282d-4d07-a3dd-725a3518ee0e">

Reply:
I mean this
<img width="603" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/11832552/9135c80a-68c3-41bb-8ce2-bf89d31d819e">

Reply:
fixed